exercises/practice/anagram/.docs/instructions.append.md

```diff
@@ -0,0 +1,3 @@
+# Implementation
+
+The Go exercise includes non-ASCII characters in the test cases, but with Go's first-class support for UTF-8, that should not bother you too much.
```

Comment:
```suggestion
Unlike other tracks, the Go version of the exercise includes test cases that use UTF-8 (non-ASCII) characters in the strings.
However, with Go's first-class support for UTF-8 given by the `rune` type, that should not bother you too much.
```

Tried to make things clearer by playing with the words a bit. Also mentioned the `rune` type as a soft hint that if the solution is leveraging runes (instead of bytes for instance), it should be ok.